gui/src/components/Router.tsx

```diff
@@ -41,6 +41,7 @@ import Navigate from "./Taipy/Navigate";
 import Menu from "./Taipy/Menu";
 import GuiDownload from "./Taipy/GuiDownload";
 import ErrorFallback from "../utils/ErrorBoundary";
+import { CircularProgress } from "@mui/material";
```

Comment:
Prefer the form
`import CircularProgress from "@mui/material/CircularProgress"`
Material UI recommends (for tree shaking I think ?)

And keep the imports grouped :-)